docs/sql-ref-literals.md

```diff
@@ -35,22 +35,19 @@ A string literal is used to specify a character string value.
 
 #### Syntax
 
-{% highlight sql %}
+```sql
 'c [ ... ]' | "c [ ... ]"
```

Comment:
@huaxingao the parameter `c` kind of looks weird especially in new format ? What do you think of character  or any_char or something like that ?

Reply:
changed to ```char```